invenio_vocabularies/jobs.py

```diff
@@ -173,4 +171,9 @@ class ImportORCIDJob(ProcessDataStreamJob):
     @classmethod
     def build_task_arguments(cls, job_obj, since=None, **kwargs):
         """Process ORCID data."""
-        return {"config": {**ORCID_PRESET_DATASTREAM_CONFIG}}
+        task_args = {"config": {**ORCID_PRESET_DATASTREAM_CONFIG}}
+        for reader in task_args["config"]["readers"]:
+            if reader["type"] == "orcid-data-sync":
```

Comment:
@ntarocco let's cross check it with your PR, maybe we could finalise the change together with what @sakshamarora1 proposes here

Reply:
@sakshamarora1 @kpsherva @palkerecsenyi do we merge this one or not?